frontend/src/locales/fr.json

```diff
@@ -29,7 +29,7 @@
     "Amount":"Montant",
     "Amount of fiat to exchange for bitcoin":"Montant de fiat à échanger contre bitcoin",
     "Invalid":"Invalide",
-    "Enter your preferred fiat payment methods. Fast methods are highly recommended.":"Entrez vos modes de paiement fiat préférées. Les modes rapides sont fortement recommandées.",
+    "Enter your preferred fiat payment methods. Fast methods are highly recommended.":"Entrez vos modes de paiement fiat préférées. Les modes rapides sont fortement recommandées."
```

Comment:
Please, make sure every line ends with "," before I merge it :) There is ~5-7 lines without it.